Testing/Unit/PowerShell/CreateReport/New-Report.Tests.ps1

```diff
@@ -3,7 +3,8 @@ Import-Module (Join-Path -Path $PSScriptRoot -ChildPath '../../../../PowerShell/
 InModuleScope CreateReport {
     Describe -Tag CreateReport -Name 'New-Report' {
         BeforeAll {
-            Mock -CommandName Write-Error {}
+            Mock -CommandName Write-Warning {}
+
```

Comment:
Is Line 7 creation on purpose?

Reply:
If I understand the question, "Do I intend to have a blank line"?
I can remove if desired.

Reply:
Oh i don't have a preference, just wanted to make sure its not an accidental blank line.